index.html

```diff
@@ -1,9 +1,106 @@
 <!DOCTYPE html>
 <html>
+<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
```

Comment:
cool, but I see that bootstrap v3 is used though v4 is latest https://www.bootstrapcdn.com/ , change this and fix if something breaks